src/components/MealCard.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import '../css/components/MealResultsListing.css';
+
+class MealCard extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {   
+    };
+  }
+
+  render() {
+    return (
+
+        <div className="singleColumn">
+          <h2 className="mealTitle">Breakfast</h2>
```

Comment:
Maybe here you could pass it some props from MealResultListing so it's easy to change from breakfast, lunch, dinner